nodejs/week1/src/backend/app.js

```diff
@@ -1,13 +1,20 @@
 const express = require("express");
```

Comment:
Hej Charles, you did not finished all the tasks from homework.md! Let me know if you need help with something!